src/atomicui/organisms/ResponsiveBottomSheet/ResponsiveBottomSheet.tsx

```diff
@@ -17,6 +17,7 @@ import {
 
 import { Flex, Loader, Text } from "@aws-amplify/ui-react";
 import { IconClose, IconNotificationBell } from "@demo/assets/svgs";
+import { ConfirmationModal } from "@demo/atomicui/molecules";
```

Comment:
nit: do we need this import then?

Reply:
Yes, since it's being used within the ResponsiveBottomSheet component.